pkg/ragengine/controllers/ragengine_controller.go

```diff
@@ -152,8 +153,34 @@ func (c *RAGEngineReconciler) applyRAG(ctx context.Context, ragEngineObj *kaitov
 
 		if err = resources.GetResource(ctx, ragEngineObj.Name, ragEngineObj.Namespace, c.Client, deployment); err == nil {
 			klog.InfoS("An inference workload already exists for ragengine", "ragengine", klog.KObj(ragEngineObj))
-			return
+			if deployment.Annotations[kaitov1alpha1.RAGEngineRevisionAnnotation] != revisionStr {
```

Comment:
What are the fields that can be updated in ragengine cr?

Reply:
Until now, everything can be updated except the nodes